spec/helpers/articles_helper_spec.rb

```diff
@@ -22,4 +22,35 @@
       expect(host).to eq "example.com"
     end
   end
+
+  describe "#image_tag_or_inline_svg" do
+    helper do
+      def internal_navigation?
```

Comment:
Because the method is comes from `helper_method` in `ApplicationController` it's not defined in this test, so when I tried to stub it I'd receiving an error saying the method wasn't there. So I defined it.